inc/customizer/settings.php

```diff
@@ -53,6 +53,34 @@ function _customize_additional_scripts( $wp_customize ) {
 add_action( 'customize_register', '_customize_additional_scripts' );
 
 /**
+ * Register settings for site identity
+ * The section title_tagline is provided by Largo under the label "Site Identity"
+ */
+function _customize_title_tagline( $wp_customize ) {
+	// Register a setting
+	$wp_customize->add_setting(
+		'site_blurb',
+		array(
+			'default'           => '',
+			'sanitize_callback' => 'wp_filter_nohtml_kses',
```

Comment:
good call - that makes sense. `esc_html()` instead, or do you have a preference?